web/src/elements/buttons/ActionButton/ak-action-button.ts

```diff
@@ -19,15 +24,14 @@ import { customElement, property } from "lit/decorators.js";
  */
 
 @customElement("ak-action-button")
-export class ActionButton extends BaseTaskButton {
+export class ActionButton extends BaseTaskButton implements IActionButton {
```

Comment:
Interfaces help Storybook (a little) understand the types it's supposed to display for controls.